cluster/vm.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: kubevirt.io/v1alpha1
+kind: VM
+metadata:
+  name: testvm
```

Comment:
Not sure about this one, I just thought about an example where you can use one or the other.

Maybe instead we should add an example folder somewhere?